implementations/rust/ockam/ockam_api/src/cloud/enroll.rs

```diff
@@ -72,24 +72,29 @@ mod node {
             dec: &mut Decoder<'_>,
         ) -> Result<Vec<u8>> {
             let req_wrapper: CloudRequestWrapper<Attributes> = dec.decode()?;
-            let cloud_address = req_wrapper.route;
+            let cloud_route = req_wrapper.route()?;
             let req_body: Attributes = req_wrapper.req;
             let req_body = RequestEnrollmentToken::new(req_body);
 
             let label = "enrollment_token_generator";
             trace!(target: TARGET, "generating tokens");
 
-            let route = self.api_service_route(&cloud_address, "enrollment_token_authenticator")?;
+            let sc = self.secure_channel(cloud_route).await?;
```

Comment:
note that this should use an identifier trust policy when we have a pinned identifier for the controller